assets/scripts/info_bubble/__tests__/DebugHoverPolygon.test.js

```diff
@@ -22,7 +22,7 @@ describe('DebugHoverPolygon', () => {
   })
 
   it('renders nothing if disabled', () => {
-    renderWithRedux(<DebugHoverPolygon />, {
+    const { container } = renderWithRedux(<DebugHoverPolygon />, {
```

Comment:
we have this pattern elsewhere..but im confused if we want to keep it or what its actually doing. but `screen.firstChild` didn't work, so it seems like there needed to be some sort of object created first?

Reply:
Yes, this is correct.